### PR TITLE
Cambios

### PR DESCRIPTION
**Pull request by Dymidless**
**Please describe the changes this PR makes and why it should be merged:**

### CODE REVIEW

File: guide/popular-topics/threads.md [hunk 1]
```diff
@@ -1,40 +1,39 @@
-# Threads
+# Hilos
 
-Threads can be thought of as temporary sub-channels inside an existing channel, to help better organize conversation in a busy channel.
+Los hilos se pueden considerar como subcanales temporales dentro de un canal existente, para ayudar a organizar mejor la conversación en un canal ocupado.
 
-## Thread related gateway events
+## Eventos de pasarela relacionados con hilos
```
Comment (socram03) [inline guide/popular-topics/threads.md:5]:
```suggestion
Eventos de Gatewat relacionados con Hilos
```

File: guide/popular-topics/threads.md [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
 
 ::: tip
-You can use the `isThread()` type guard to make sure a channel is a ThreadChannel!
+¡Puedes usar la protección de tipo `isThread()` para asegurarte de que un canal sea un Canal de hilo!
 :::
 
-Threads introduce a number of new gateway events, which are listed below:
+Los hilos introducen una serie de nuevos eventos de puerta de enlace, que se enumeran a continuación:
```
Comment (socram03) [inline guide/popular-topics/threads.md:11]:
```suggestion
Los hilos introducen una serie de nuevos eventos de Gateway, que se enumeran a continuación:
```

**Review by socram03 — CHANGES_REQUESTED** [review]
Palabras claves se dejan en su idioma original

**Comment by Dymidless** [comment]
Listo

**Review by Awoocado — APPROVED** [review]

**Pull request merged**